esp/src/src-react/components/InfoGrid.tsx

```diff
@@ -236,6 +236,14 @@ export const InfoGrid: React.FunctionComponent<InfoGridProps> = ({
         }
     }, [data.length]);
 
+    React.useEffect(() => {
+        if (filterCounts.error === 0 && filterCounts.warning === 0 && filterCounts.cost === 0) {
+            setInfoChecked(true);
+        } else {
+            setInfoChecked(false);
+        }
+    }, [filterCounts]);
```

Comment:
When a job is running, this might get repeatedly set?

Reply:
I executed a couple of scripts that ran for 20 or 30 seconds, with a breakpoint set at the top of this useEffect() and that did not seem to be the case. The breakpoint triggered once when I loaded into the WUDetails page, and then again when the WU finished.